Tests/Model/InteractionTestCase.swift

```diff
@@ -24,7 +24,7 @@ class InteractionTestCase: XCTestCase {
 	var builder: PactBuilder!
 
 	private var pactDirectory: String {
-		ProcessInfo.processInfo.environment["PACT_OUTPUT_DIR"]!
+		ProcessInfo.processInfo.environment["PACT_OUTPUT_DIR"] ?? "/tmp/pacts"
```

Comment:
thought: I'm personally a fan of force-unwrapping in tests. But only in tests. Why? Fail fast. If my test crashes it's most likely "developer issue" and should be looked into much deeper.

Reply:
That force unwrap is finding `nil` for me. What should I be doing differently?

Reply:
Ah of course, that thing is a bit obscure in PactSwift's 
![scheme build config](https://github.com/surpher/PactSwift/blob/main/Documentation/images/12_xcode_scheme_env_setup.png?raw=true)

If that doesn't work, great, we can find a setup that works across contributors' systems. 
I'm digging into re-introducing Linux support with adding Swift Package Manager Plugin to pull down a `.so` from pact-foundation/pact-reference releases. 

There definitely was an interface that would take `URL`:
```swift
MockService(
    consumer: "consumer",
    provider: "provider",
    writePactTo: URL(fileURLWithPath: "/absolute/path/pacts/folder", isDirectory: true)
)
```

But I'm not quite sure anymore if it's still there. Like I said, I haven't used PactSwift on a project in more than a year 😞